src/wpantund/NCPControlInterface.h

```diff
@@ -284,6 +284,12 @@ class NCPControlInterface {
 
 	boost::signals2::signal<void(const EnergyScanResultEntry&)> mOnEnergyScanResult;
 
+public:
+	// ========================================================================
+	// Network time-related Member Functions
+
+	boost::signals2::signal<void(const NetworkTimeUpdate&)> mOnNetworkTimeUpdate;
```

Comment:
How about changing this signal format to use `ValueMap` as input param instead?
```
boost::signals2::signal<void(const ValueMap &)> mOnNetworkTimeUpdate;
```
This will be more flexible and would help later with preparing the DBus message.